### PR TITLE
KB-15-Missing-site-icon-favicon

### PR DESCRIPTION
**Pull request by Markusplay**

### CODE REVIEW

**Review by niravzi — CHANGES_REQUESTED** [review]
These changes are insufficient.
Please take the icons from attached archive:
[favicon.zip](https://github.com/user-attachments/files/17458072/favicon.zip)

Replicate the changes from schedule project from this branch:  https://github.com/kpi-ua/schedule.kpi.ua/pull/246

**Comment by Markusplay** [comment]
image

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -23,13 +23,18 @@ export default function Intellect({ Component, pageProps }: AppPropsWithLayout)
...
             <Head>
-                <meta charSet="utf-8" />
+                <meta charSet="utf-8"/>
                 <title key="title">Інтелект | Викладачі та науковці</title>
 
-                <link rel="shortcut icon" type="image/vnd.microsoft.icon" href='/favicon.ico' />
-                <link rel="preconnect" href="https://fonts.googleapis.com" />
-                <link rel="preconnect" href="https://fonts.gstatic.com" crossOrigin="anonymous" />
-
+                <link rel="shortcut icon" type="image/vnd.microsoft.icon" href='/favicon.ico'/>
```
Comment (niravzi) [inline src/pages/_app.tsx:29]: Duplicate

**Review by niravzi — CHANGES_REQUESTED** [review]

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -23,13 +23,18 @@ export default function Intellect({ Component, pageProps }: AppPropsWithLayout)
...
         <>
             <Head>
-                <meta charSet="utf-8" />
+                <meta charSet="utf-8"/>
                 <title key="title">Інтелект | Викладачі та науковці</title>
 
-                <link rel="shortcut icon" type="image/vnd.microsoft.icon" href='/favicon.ico' />
-                <link rel="preconnect" href="https://fonts.googleapis.com" />
-                <link rel="preconnect" href="https://fonts.gstatic.com" crossOrigin="anonymous" />
+                <link rel="shortcut icon" type="image/vnd.microsoft.icon" href='/favicon.ico'/>
```
Comment (niravzi) [inline src/pages/_app.tsx:29]: Order of links matters.
Please copy the changes from schedule in the same order and with the same attributes as there.

**Review by niravzi — CHANGES_REQUESTED** [review]

File: src/pages/_app.tsx (hunk 2 above)
Comment (Markusplay) [reply src/pages/_app.tsx]: ordered icon links as it is done in schedule

**Review by niravzi — APPROVED** [review]

**Pull request merged**